pkgs/top-level/go-packages.nix

```diff
@@ -878,10 +878,11 @@ let
   };
 
   git-lfs = buildFromGitHub {
-    rev = "v1.0.0";
+    date = "1.1.1";  # "date" is effectively "version"
```

Comment:
FYI this is here to avoid breaking nix-env's version string parsing, which would otherwise make it difficult to uninstall the package later on.